bar-trender-front-end/src/components/ApiFormSubmit.js

```diff
@@ -28,19 +32,26 @@ class POSTForm extends Component {
         this.setState({ [e.target.name]: e.target.value });
     }
 
-    handleSearch() {
+    async handleSearch() {
         // this.props.searchEngine(this.state.term);
-
-        alert("A form was submitted: " + JSON.stringify(this.state));
-
-        // fetch('https://your-node-server-here.com/api/endpoint', {
-        //     method: 'POST',
-        //     // We convert the React state to JSON and send it as the POST body
-        //     body: JSON.stringify(this.state)
-        //   }).then(function(response) {
-        //     console.log(response)
-        //     return response.json();
-        //   });
+        
+        //alert("A form was submitted: " + JSON.stringify(this.state));
+        this.state = {
+            "filters":{
+            }
+        };
+        await fetch('http://localhost:8000/v1/search/establishments', {
+            method: 'POST',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7138516)
```suggestion
            method: "POST",
```